model.py

```diff
@@ -9262,6 +9278,53 @@ def set_protocol(self, new_protocol):
         for child in self.children:
             child.protocol = new_protocol
 
+    # TODO: The default loan period needs to be expanded to handle
+    # different defaults for different media types (e.g. on Overdrive
+    # the default loan period for audiobooks is 14 days, and for video
+    # it's 5 days). This isn't a high priority because the license
+    # source generally tells us when each loan will end.
```

Comment:
It's important for hold time estimates, not that we do a good job of them even with this.